website/docs/workflows/reference/workflow-triggers.md

```diff
@@ -0,0 +1,82 @@
+---
+description: This page provides detailed information on workflow triggers available in Appsmith.
+title: Workflow Triggers  
+hide_title: true  
+toc_min_heading_level: 2  
+toc_max_heading_level: 3  
+---
+
+<!-- vale off -->
+
+<div className="tag-wrapper">
+ <h1>Workflow Triggers</h1>
+
+<Tags
+tags={[
+{ name: "Business", link: "https://www.appsmith.com/pricing", additionalClass: "business" }
+]}
+/>
+
+</div>
+
+<!-- vale on -->
+
+Workflow triggers start workflow runs based on specific conditions or events. This page provides information about the workflow triggers available in Appsmith, including their configurations, parameters, and usage examples.
+
+## Webhook trigger
+
+A webhook trigger allows external services to start a workflow by sending an HTTP request to a specific URL. This trigger is useful for integrating with applications and services that can send HTTP requests when certain events occur. For example, you can set up a webhook trigger to automatically start a workflow whenever a user signs up on your website, allowing you to send a welcome email or update a database in real time.
```

Comment:
We should call it POST request instead of HTTP as we support HTTPS and HTTP might come off as insecure.